vinca_linux_64.yaml

```diff
@@ -6,9 +6,9 @@ conda_index:
   - packages-ignore.yaml
 
 # Reminder for next full rebuild, the next build number should be 4
```

Comment:
Could you please change this comment to `should be 6` here and in the other `vinca_*` files?

Reply:
@Tobias-Fischer updated.